readwise.py

```diff
@@ -51,11 +51,11 @@ def __post_init__(self):
 
 
 class ReadwiseClient:
-    def __init__(self, api_key: str=None):
+    def __init__(self, api_key: str=None, latest_fetch_time=None):
```

Comment:
Nit: I think it makes more sense to have a `set_latest_fetch_time()` function (that returns `self` so it can be chained), because the parameter isn't needed and can be changed in the future. Then we should rename `self.latest_fetch_time` to `self._latest_fetch_time`.